intel_extension_for_transformers/neural_chat/tools/embedding_finetune/README.md

```diff
@@ -0,0 +1,139 @@
+# Finetune Embedding Model on Task-Specific Datasets
+
+## 1. Introduction
+In this example, we show how to construct the training data for finetuning the embedding model and finetuning the specific embedding model.
+
+## 2. Requirements
+* **On CPU**
+```
+pip install -r requirements_cpu.txt
+```
+
+* **On GPU**
+```
+pip install -r requirements_gpu.txt
```

Comment:
This gpu is cuda, right? It's better to rename to requirements_cuda.txt. It will confuse with Intel GPU.